apps/site/navigation.json

```diff
@@ -77,6 +77,11 @@
       "icon": "linkedin",
       "link": "https://www.linkedin.com/company/node-js",
       "alt": "LinkedIn"
+    },
+    {
+      "icon": "bluesky",
+      "link": "https://bsky.app/profile/nodejs.org",
+      "alt": "Bluesky"
     }
```

Comment:
IMO Bluesky should go next to Twittter/X, as they are similar platforms (in terms of how they work)

Reply:
<img width="477" alt="image" src="https://github.com/user-attachments/assets/db01fc1d-79f9-4226-bf57-907de033cb67">
done